doc/build-unix.md

```diff
@@ -1,30 +1,9 @@
-UNIX BUILD NOTES
+UNIX Build Instructions and Notes
```

Comment:
As said on #134 this line could perhaps be upstreamed to https://github.com/bitcoin/bitcoin ?